queries.md

```diff
@@ -5,28 +5,65 @@
 ## 1. Find all the companies that include 'Facebook' on the **name** field.
 
  - **`query`**: {name: 'Facebook'}
+ - **`query`**: 
+- **`projection`**: 
  
  ## 2. Find all the companies which **category_code** is 'web'. Retrive only their `name` field:
 
  - **`query`**: {category_code: 'web'}
  - **`projection`**: {name: 1, _id: 0}
 
 ## 3. Find all the companies named "Twitter", and retrieve only their `name`, `category_code` and `founded_year` fields.
+- **`query`**: {name: "Twitter"}
+- **`projection`**: {name: 1, category_code:1, founded_year:1}
 
 ## 4. Find all the companies who have `web` as their **category_code**, but limit the search to 50 companies.
+- **`query`**: {category_code: "web"}
+- **`skip`**: 0
+- **`limit`**: 50
 
 ## 5. Find all the companies which **category_code** is 'enterprise' and have been founded in 2005. Retrieve only the `name`, `category_code` and `founded_year` fields.
+ - **`query`**: {$and: [{category_code: "enterprise"}, {founded_year: 2005}]}
+- **`projection`**: {name: 1, category_code:1, founded_year:1}
 
 ## 6. Find all the companies that have been **founded** on the 2000 or have 20 **employees**. Sort them descendingly by their `number_of_employees`.
 
+ - **`query`**: {$or: [{founded_year: 2000}, {number_of_employees: 20}]}
+- **`sort`**: {number_of_employees: -1}
+
+
 ## 7. Find all the companies that do not include `web` nor `social` on their **category_code**. Limit the search to 20 documents and retrieve only their `name` and `category_code`.
+ - **`query`** (solution 1): {$and: [{category_code: {$ne: "web"}}, {category_code: {$ne: "social"}}]}
+ - **`query`** (solution 2, far better!):  {$nor:[{category_code:"web"},{category_code:"social"}]}
```

Comment:
Haha yes! Better indeed, but there's even better yet: **$nin**
```javascript
+{category_code: { $nin: [ "web", "social"] }}
```
This works only because the values you want to exclue both are from "category_code".

Still, big up for the use of **$nor** ;)